nova/tests/unit/compute/test_compute_api.py

```diff
@@ -6172,7 +6179,7 @@ def test_get_all_build_requests_decrement_limit(self,
                 sort_keys=['baz'], sort_dirs=['desc'])
             fields = ['metadata', 'info_cache', 'security_groups']
             mock_inst_get.assert_called_once_with(
-                self.context, {'foo': 'bar'}, limit=8, marker=None,
+                self.context, {'foo': 'bar'}, limit=10, marker=None,
                 fields=fields, sort_keys=['baz'], sort_dirs=['desc'])
```

Comment:
From the name of the test, it looks like it's testing something that we don't do anymore. Shouldn't we remove it instead?

Reply:
Or maybe we should rename it, so that's clear, what we're testing here.